metron-platform/metron-indexing/metron-indexing-storm/src/test/java/org/apache/metron/indexing/integration/HDFSIndexingIntegrationTest.java

```diff
@@ -151,11 +152,11 @@ public String cleanField(String field) {
 
   @Override
   public String getTemplatePath() {
-    return "../metron-indexing/src/main/config/hdfs.properties.j2";
+    return "../metron-indexing-common/src/main/config/hdfs.properties.j2";
   }
 
   @Override
   public String getFluxPath() {
-    return "../metron-indexing/src/main/flux/indexing/batch/remote.yaml";
+    return "../metron-indexing-common/src/main/flux/indexing/batch/remote.yaml";
```

Comment:
Should there be Storm artifacts in *-common?  Probably should be metron-indexing-storm, and will impact the metron.spec as well.